internal/provider/data_source_aci_route_control_profile_test.go

```diff
@@ -28,7 +28,6 @@ func TestAccDataSourceRtctrlProfileWithFvTenant(t *testing.T) {
 					resource.TestCheckResourceAttr("data.aci_route_control_profile.test", "owner_key", ""),
 					resource.TestCheckResourceAttr("data.aci_route_control_profile.test", "owner_tag", ""),
 					resource.TestCheckResourceAttr("data.aci_route_control_profile.test", "route_control_profile_type", "combinable"),
-					resource.TestCheckResourceAttr("data.aci_route_control_profile.test", "route_map_continue", "no"),
```

Comment:
Why are we removing these?

Reply:
Introduced version_mismatch in default which takes care of this.